harvard-lts/drs-translation-service
# Etd 239: ETDTranslateDataStructureService

**TranslateDataStructureService class for the ETDs**
* * *

**GitHub Issue**: https://jira.huit.harvard.edu/browse/ETD-239
https://jira.huit.harvard.edu/browse/ETD-262


# What does this Pull Request do?
This creates a TranslateDataStructureService class for the ETDs.  The ETDTranslateDataStructureService class calls the ContentModelMappingBuilder() to do the directory formatting.

# How should this be tested?

A description of what steps someone could take to:

0.  copy `env-template.txt` to `.env` and set to dev if you have not already done so
1.  start up docker `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
2.  enter container `docker exec -it dts bash`
3. run test `pytest ./tests/unit/test_translate_data_structure_service.py`  all tests should pass.
4.  stop container `docker-compose -f docker-compose-local.yml down`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? n

# Interested parties
Tag (@ mention) interested parties @awoods
